install/ui/src/freeipa/serverconfig.js

```diff
@@ -85,7 +85,10 @@ return {
                                 { label: '@i18n:authtype.type_disabled', value: 'disabled' },
                                 { label: '@i18n:authtype.type_password', value: 'password' },
                                 { label: '@i18n:authtype.type_radius', value: 'radius' },
-                                { label: '@i18n:authtype.type_otp', value: 'otp' }
+                                { label: '@i18n:authtype.type_otp', value: 'otp' },
+                                { label: '@i18n:authtype.type_pkinit', value: 'pkinit' },
+                                { label: '@i18n:authtype.type_spake', value: 'spake' },
+                                { label: '@i18n:authtype.type_fast', value: 'fast' }
```

Comment:
You are introducing here i18n labels for each new type but don't add them in `ipaserver/plugins/internal.py`. Please add them there.